source/compatibility.txt

```diff
@@ -50,5 +50,5 @@ The first column lists the driver version.
 
 .. include:: /includes/language-compatibility-table-pymongo.rst
 
-For more information on how to read the compatibility tables, see our guide on 
+For more information about how to read the compatibility tables, see
```

Comment:
I think this should be moved up above the "Python 2" header so that it's closer to the actual table (since the Python 2 section doesn't have a table in it)